src/Whoops/Handler/PlainTextHandler.php

```diff
@@ -151,14 +171,18 @@ public function setTraceFunctionArgsOutputLimit($traceFunctionArgsOutputLimit)
     public function generateResponse()
     {
         $exception = $this->getException();
-        return sprintf(
-            "%s: %s in file %s on line %d%s\n",
-            get_class($exception),
-            $exception->getMessage(),
-            $exception->getFile(),
-            $exception->getLine(),
-            $this->getTraceOutput()
-        );
+        $message = $this->getExceptionOutput($exception);
+
+        if ($this->addPreviousToOutput) {
+            $previous = $exception->getPrevious();
+            while ($previous) {
+                $message .= "\nBefore: " . $this->getExceptionOutput($previous);
```

Comment:
```suggestion
                $message .= "\nPrevious: " . $this->getExceptionOutput($previous);
```

Reply:
updated with a line break and caused by information...